web/packages/shared/components/AuthorizeDeviceWeb/AuthorizeDeviceWeb.tsx

```diff
@@ -36,10 +36,13 @@ import { makeDeepLinkWithSafeInput } from 'shared/deepLinks';
 
 export const PassthroughPage = () => {
   const ctx = useTeleport();
+  const { search } = useLocation();
   const { id, token } = useParams<{
     id: string;
     token: string;
   }>();
+  const redirect = new URLSearchParams(search).get('redirect_uri');
+
   const { cluster, username } = ctx.storeUser.state;
   const deviceTrustAuthorize = makeDeepLinkWithSafeInput({
```

Comment:
The comment for this function says:

> `makeDeepLinkWithSafeInput` creates a deep link by interpolating passed arguments.
> Important: This function does not perform any validation or parsing. It must not be called with user-generated content.

This comment is no longer valid. IIRC my original intention, back when we didn't have search params here, was that `proxyHost`, `path` and `username` were simply concatenated together. So if we passed user-generated content there, it'd be possible to manipulate the function to generate some weird URLs.

With search params this is not a concern, since each param and its value get individually encoded. Could you update the comment so that it's up to date?

Reply:
I'm unsure your expected outcome for this comment to make it up to date besides calling "this must not be called with user-generated content". I removed that since I'm passing in user generated content now but if I'm missing the mark here, let me know.